experimental/dotnet/BUILD

```diff
@@ -1,11 +1,10 @@
 package(default_visibility = ["//visibility:public"])
 
+load("//base:distro.bzl", "DISTRO_SUFFIXES")
 load("@io_bazel_rules_docker//container:container.bzl", "container_image")
 load("@package_bundle_amd64_debian9//file:packages.bzl", packages_amd64_debian9 = "packages")
 load("@package_bundle_amd64_debian10//file:packages.bzl", packages_amd64_debian10 = "packages")
 
-DISTRO_SUFFIXES = ("_debian9", "_debian10")
-
 DISTRO_PACKAGES = {
```

Comment:
should distropackages be built off of distrosuffixes?

Reply:
Ah, good catch. Just realized these can go away too.